packages/cashier/build/constants.js

```diff
@@ -28,6 +28,7 @@ const ALIASES = {
     Stores: path.resolve(__dirname, '../src/stores'),
     Types: path.resolve(__dirname, '../src/types'),
     Utils: path.resolve(__dirname, '../src/utils'),
+    '@deriv/hooks': path.resolve(__dirname, '../src/hooks'),
```

Comment:
@farzin-deriv Are you using `@deriv/hooks` to be prepared for having a separate package for Hooks? :grin:

Reply:
@Bahar-FS Yes, I hope we can extract them from the cashier into a separate package later 😅